packages/core/src/App/AppContent.tsx

```diff
@@ -22,6 +24,11 @@ import Routes from './Containers/Routes/routes.jsx';
 import Devtools from './Devtools';
 import initDatadog from '../Utils/Datadog';
 
+const gb = new GrowthBook({
```

Comment:
Let's define a descriptive name for this instance.
Ref: https://github.com/airbnb/javascript?tab=readme-ov-file#naming--descriptive

```suggestion
const growthbook = new GrowthBook({
```